torchtune/data/_prompt_templates.py

```diff
@@ -110,11 +110,13 @@ def __call__(
             if message.role in self.template:
                 prepend_tag = self.template[message.role][0]
                 append_tag = self.template[message.role][1]
-                content = (
-                    [{"type": "text", "content": prepend_tag}]
-                    + message.content
-                    + [{"type": "text", "content": append_tag}]
-                )
+                content = {message.content}
```

Comment:
the {} should be removed here, otherwise we cannot concatenate lists below. otherwise the rest looks good

Reply:
Fixed in both cases